game-core/src/main/java/games/strategy/triplea/help/HelpSupport.java

```diff
@@ -1,30 +1,51 @@
 package games.strategy.triplea.help;
 
-import java.io.BufferedReader;
+import static com.google.common.base.Preconditions.checkNotNull;
+
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
+import java.util.logging.Level;
+
+import javax.annotation.Nullable;
+
+import org.apache.commons.io.IOUtils;
+
+import com.google.common.html.HtmlEscapers;
+
+import lombok.extern.java.Log;
 
 /**
- * A class for loading help files from the data folder (merged with src at runtime).
+ * A class for loading help files from the classpath.
  */
-public class HelpSupport {
+@Log
+public final class HelpSupport {
+  private HelpSupport() {}
+
+  /**
+   * Returns the HTML help text stored in the specified file. The file is assumed to be located on the classpath within
+   * the {@code games.strategy.triplea.help} package.
+   */
   public static String loadHelp(final String fileName) {
-    try {
-      final InputStream is = HelpSupport.class.getResourceAsStream(fileName);
-      final BufferedReader br = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8));
-      String line;
-      final StringBuilder sb = new StringBuilder();
-      while ((line = br.readLine()) != null) {
-        sb.append(line);
+    checkNotNull(fileName);
+
+    try (@Nullable
+    InputStream is = HelpSupport.class.getResourceAsStream(fileName)) {
```

Comment:
An unfortunate consequence of the Eclipse formatter.

Reply:
I didn't even know try with resource allows null in the sense that it doesn't throw an exception when null is used.

Reply:
Yeah, if you look at the generated bytecode, you'll see that it checks to see if the resource is `null` in the equivalent of the `finally` block before attempting to close the resource.